SamplifyAPIClientTest/SamplifyClientTest.cs

```diff
@@ -3,13 +3,16 @@
 using System.Net.Http;
 using Dynata.SamplifyAPIClient;
 using Xunit;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
 
 namespace SamplifyAPIClientTest
 {
     public class SamplifyClientTest
-    {
+    { 
         [Fact]
-        public void TestAuth()
+        public void TestAth()
```

Comment:
Probably just typo. Auth is better. (again, public repo. and all)

Reply:
Fixed.